src/ProductCard.js

```diff
@@ -139,28 +146,29 @@ const ButtonBack = styled.button`
 const Logo = styled.img`
   margin: 20px 0 0 30px;
   grid-area: logo;
-  height: 10rem;
+  height: 8rem;
 `
 
 const Text = styled.p`
   width: 200px;
-  margin: 40px 20px 0 -125px;
+  margin: 40px 20px 0 -180px;
   grid-area: portrait;
 `
 
 const Headline = styled.h2`
   grid-area: details;
   padding-left: 50px;
-  margin: -10px;
+  margin: -40px -10px; ;
 `
 
-const Details = styled.ul`
+const Details = styled.ol`
+  margin: -10px 15px;
   grid-area: details;
 `
 
 const List = styled.li`
```

Comment:
This should not be called `List` as it is not a list but a list item. So please call it `ListItem` or something like that.